<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1625</width>
    <height>794</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1800</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="addonConfig">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>130</y>
      <width>261</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Add-On Configuration</string>
    </property>
    <widget class="QGroupBox" name="wallConfig">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>280</y>
       <width>141</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Explorer Mode</string>
     </property>
     <widget class="QPushButton" name="startExplore">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopExplore">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="movementKeys">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>241</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Movement Keys</string>
     </property>
     <widget class="QPushButton" name="up">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>93</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>UP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="right">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>93</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>RIGHT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="down">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>93</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>DOWN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="left">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>93</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>LEFT</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="field">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>1301</width>
      <height>751</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1280</width>
      <height>720</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1500</width>
      <height>900</height>
     </size>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fpsCounterLabel">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS: XX </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
